src/ucs/datastruct/lru.h

```diff
@@ -0,0 +1,127 @@
+/**
+* Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001-2023. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef __LRU_H__
+#define __LRU_H__
+
+#include <stddef.h>
+#include <stdint.h>
+
+/* Needed by array.inl so must be first */
+#include <ucs/type/status.h>
+
+#include <ucs/datastruct/array.inl>
+#include <ucs/datastruct/khash.h>
+#include <ucs/debug/assert.h>

```

Comment:
why needed?

Reply:
ucs_assert(ret != UCS_KH_PUT_FAILED);  -> in ucs_lru_put